src/components/avatar-picker/AvatarPicker.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component, Fragment, createRef } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import Avatar from '../avatar';
+import { CameraIcon, PlusIcon, UserIcon } from '../icon';
+
+import styles from './AvatarPicker.css';
+
+class AvatarPicker extends Component {
+    state = {
+        image: undefined,
+        imageURL: undefined,
+    };
+
+    inputRef = createRef();
+
+    render() {
+        const { label, className, onChange, ...rest } = this.props;
+
+        return (
+            <Fragment>
+                { this.renderInput() }
+                <div className={ classNames(styles.container, className) } onClick={ this.handleAvatarLoaderClick }>
+                    <div className={ styles.circleWrapper }>
+                        { this.renderPlusIcon() }
+                        { this.renderAvatar(rest) }
+                    </div>
+                    <label className={ styles.label }>{ label }</label>
+                </div>
+            </Fragment>
+        );
+    }
+
+    renderInput = () => (
+        <input
+            ref={ this.inputRef }
+            className={ styles.hidden }
+            type="file"
+            accept="image/*"
+            onChange={ this.handleOnChange } />
+    );
+
+    renderPlusIcon = () => {
+        if (!this.props.imageURL && !this.state.imageURL) {
+            return (
+                <div className={ styles.smallCircle }>
+                    <PlusIcon className={ styles.icon } />
+                </div>
+            );
+        }
+
+        return (
+            <div className={ styles.imageOverlay }>
+                <CameraIcon className={ styles.cameraIcon } />
+            </div>
+        );
+    };
+
+    renderAvatar = (props) => {
```

Comment:
Avoid arrow functions whenever possible to use standard functions. Please change here and check the rest of the code.

Reply:
Can't change that function because I'm accessing `this` inside. But I'll check other occurrences.